### PR TITLE
Fresh run at the introduction section

### PR DESCRIPTION
**Pull request by zachernuk**
Re-framed the scope of what a declarative element being managed directly by the user agent actually has the ability to achieve!

***
<a href="https://pr-preview.s3.amazonaws.com/immersive-web/model-element/pull/88.html" title="Last updated on Jul 16, 2024, 4:13 AM UTC (885cbf0)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/immersive-web/model-element/88/1b25a02...885cbf0.html" title="Last updated on Jul 16, 2024, 4:13 AM UTC (885cbf0)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -59,6 +59,31 @@ <h1 id="title">
...
+    <section>
+      <h2>
+        Introduction
+      </h2>
+      <p>
+        The HTML &lt;model&gt; element allows a website to embed interactive 3D
+        models as conveniently as any other visual media. Models are served as 
+        a standalone resource. The &lt;model&gt; element also has support for 
+        interaction and animation playback while presented within the page, 
+        and supports more immersive experiences, such as the stereoscopic
```
Comment (LaszloGombos) [inline index.html:71]: Perhaps suggest that this is stereoscopic/immersive is optional if the availble.

Comment (zachernuk) [reply index.html]: Ah yes - I have now updated with the language we had actually gotten through internally!

File: index.html [hunk 2]
```diff
@@ -59,6 +59,31 @@ <h1 id="title">
...
+        and supports more spatial experiences, such as the stereoscopic 
+        display of 3D content where available.
+      </p>
+      <p>
+        As &lt;model&gt; is rendered directly by the user agent, it has
+        the ability to utilize privileged information that is not made
+        available to a Javascript context, such as the user's head and eye
+        position or the lighting of the user's environment. Additionally, the
+        use of accessibility information and controls may be engaged in without
+        divulging that information to a Javascript context.
```
Comment (marcoscaceres) [inline index.html:80]: We should probably frame this as: 

```suggestion
        the ability to utilize privileged information, such as the user's head and eye
        position or the lighting of the user's environment, without exposing that information to script. 
        Additionally, the use of accessibility information and controls can be engaged in a privacy-preserving
        manner.
```

File: index.html [hunk 3]
```diff
@@ -59,6 +59,30 @@ <h1 id="title">
...
+        models as conveniently as any other visual media. Models are served as 
+        a standalone resource. The &lt;model&gt; element also has support for 
+        interaction and animation playback while presented within the page, 
+        and supports more spatial experiences, such as the stereoscopic 
+        display of 3D content where available.
+      </p>
+      <p>
+        As &lt;model&gt; is rendered directly by the user agent, it has
+        the ability to utilize privileged information, such as the user's head and eye
+        position or the lighting of the user's environment, without exposing that information to script.
```
Comment (LaszloGombos) [inline index.html:77]: "to script" -->  "to JavaScript"

Comment (zachernuk) [reply index.html]: good call. Updated!

Comment (LaszloGombos) [reply index.html]: thx

**Review by LaszloGombos — APPROVED** [review]
LGTM

**Review by marcoscaceres — APPROVED** [review]
lgtm

**Pull request merged**